x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -64,23 +117,36 @@ export const AlertIndex = memo(() => {
       }
       return null;
     };
-  }, [json]);
+  }, [json, pageSize, total]);
+
+  const pagination = useMemo(() => {
+    return {
+      pageIndex,
+      pageSize,
+      pageSizeOptions: [10, 50, 100],
```

Comment:
We should all sync up on the values to use for page size. @parkiino and I are using `[10, 20, 50]`, which I think we discussed with Bonnie.